pkg/helm/helm.go

```diff
@@ -80,6 +84,18 @@ func (h *Hydrator) templateArgs(ctx context.Context, destDir string) ([]string,
 	return args, nil
 }
 
+func (h *Hydrator) appendValuesArgs(args []string) ([]string, error) {
+	var values map[string]interface{}
```

Comment:
For the `values` key/value pairs, what type(s) can the value be? I expected it would only be `string`

Reply:
value can be different types, string, integer, bool ...